test/tools/libtesteth/ImportTest.cpp

```diff
@@ -292,7 +291,7 @@ std::tuple<eth::State, ImportTest::ExecOutput, eth::ChangeLog> ImportTest::execu
         if (!removeEmptyAccounts)
         {
             // Touch here bacuse coinbase might be suicided above
-            initialState.addBalance(_env.author(), 0);  // touch coinbase no matter what
+            initialState.addBalance(_env.author(), 0);  // imitate mining reward
```

Comment:
Why is this done again?  Before, this is a line
```
initialState.addBalance(_env.author(), 0);  // imitate mining reward
```

Reply:
because suicide coibase will delete coinbase account from the state. we need to touch it again after suicide.